snapcraft.yaml

```diff
@@ -650,6 +650,32 @@ parts:
       - bin/nvidia-container-cli*
       - lib/libnvidia-container*.so*
 
+  nvidia-container-toolkit:
+    source: https://github.com/NVIDIA/nvidia-container-toolkit
+    source-depth: 1
+    source-tag: v1.15.0
+    source-type: git
+    build-snaps:
+      - go
+    plugin: make
+    override-prime: |-
+      [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && exit 0
```

Comment:
Does this only work on amd64 and arm64 then?

Reply:
Actually, it seems to also be supported for `ppc64le`

Reply:
its just Go right?

Reply:
I'm fine with limiting it if you think there is a reason that this tool wouldn't be used on those architectures, even if it can be built. Can iGPUs be used on ppc64le and s390x?

Reply:
No idea.. I'm asking.

Reply:
> its just Go right?

No, there is a small C lib here: https://github.com/NVIDIA/nvidia-container-toolkit/blob/a470818ba7d9166be282cd0039dd2fc9b0a34d73/internal/dxcore/dxcore.c#L7

Reply:
I asked @elezar

Reply:
I'm OOTO. Will get to this on Tuesday.

Reply:
Note that the linked c file should not be relevant on these platforms either since these are specific to discovering the location of driver libraries on WSL2-based systems. If the dxcore.so shared libraries are not present, these are never used.

Reply:
Thanks @elezar 

@gabrielmougard are you able to address this please?